packages/prio3/src/prio3.ts

```diff
@@ -267,7 +267,7 @@ export class Prio3<Measurement> implements Prio3Vdaf<Measurement> {
     return { verifier, jointRand };
   }
 
-  private encodePrepareMessage(
+  private encodePrepareState(
```

Comment:
This should keep the name `encodePrepareMessage`. (Technically, we are using this function to encode both prepare message shares and prepare messages, but these have the same concrete type and encoding in the context of Prio3)